fuzz/build.info

```diff
@@ -193,6 +198,10 @@ IF[{- !$disabled{tests} -}]
   INCLUDE[decoder-test]=../include
   DEPEND[decoder-test]=../libcrypto
 
+  SOURCE[acert-test]=acert.c test-corpus.c fuzz_rand.c
```

Comment:
Doesnt look like this requires either fuzz_rand.c or ../libssl

Reply:
Thank you.  Yes, neither are used in this test case.  Removed.